rest_framework/pagination.py

```diff
@@ -534,7 +540,7 @@ def get_next_link(self):
         if not self.has_next:
             return None
 
-        if self.cursor and self.cursor.reverse and self.cursor.offset != 0:
+        if self.cursor and self.cursor.reverse and self.cursor.offset != 0 and self.page:
```

Comment:
Demonstrate different issues in different pull requests? Yes, please.

Not because I'm being an ass, but because I have to deal with 100's, 1000's of issues & PRs - keeping things properly separated means we can progress things quickly, itemize change lists properly, and don't get blocked on one contentious part of an issue being reviewed, while another completely separate and valid part could be merged.